halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Providers.java

```diff
@@ -46,16 +51,35 @@
   GoogleProvider google = new GoogleProvider();
   KubernetesProvider kubernetes = new KubernetesProvider();
   OpenstackProvider openstack = new OpenstackProvider();
+  @JsonProperty(access = Access.WRITE_ONLY)
```

Comment:
If this is write only, and someone has the written in their halconfig, how does halyard read it to do the merge?

Reply:
According to JsonProperty.Access javadoc:  WRITE_ONLY means that the property may only be written (set) for deserialization, but will not be read (get) on serialization, that is, the value of the property is not included in serialization.
So, the existing "oraclebmcs" will be deserialized into PersistenceStorage and will be consolidated with the deserialized "oracle" to produce a new "oracle" for serialization.  The serialized "persistentStorage" will not contain "oraclebmcs".

Reply:
Nice! Thanks